bcda/models/mock_repository_test.go

```diff
@@ -0,0 +1,110 @@
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
This is auto-generated?  Should we provide instructions somewhere in our `README` to inform developers how to generate these files, in order to ensure satisfactory maintainability?

Reply:
Great idea, let update the readme with this information